index.js

```diff
@@ -881,12 +881,12 @@ RingPop.prototype.handleOrProxyAll =
                     url: req && req.url,
                     dest: dest
                 });
-                self.proxyReq({
+                self.proxyReq(_.defaults({
                     keys: keys,
                     req: req,
```

Comment:
`keys` and `req` are already in `opts`, so you could leave them out here if you don't want to set them twice